content/backstage/plugins/sonarqube.md

```diff
@@ -63,12 +64,19 @@ gettingStarted:
         target: https://your.sonarqube.instance.com/api
         allowedMethods: ['GET']
         headers:
-          Authorization: # Content: 'Basic base64("<api-key>:")' <-- note the trailing ':' # Example: Basic bXktYXBpLWtleTo=
-            $env: SONARQUBE_AUTH_HEADER
+          Authorization: Basic ${SONARQUBE_AUTH}
+          # Environmental variable: SONARQUBE_AUTH
+          # Value: 'base64("<sonar-auth-token>:")'
+          # Encode the "<sonar-auth-token>:" string using base64 encoder.
+          # Note the trailing colon (:) at the end of the token.
+          # Example environmental config: SONARQUBE_AUTH=bXktYXBpLWtleTo=
+          # Fetch the sonar-auth-token from https://sonarcloud.io/account/security/
```

Comment:
Same in this section. Can we make some more understandable instructions for the reader?

Reply:
How does this look?